package.json

```diff
@@ -19,24 +19,24 @@
   },
   "homepage": "https://github.com/PiwikPRO/ngx-piwik-pro#readme",
   "dependencies": {
-    "@angular/animations": "~13.2.0",
-    "@angular/cdk": "^13.2.3",
-    "@angular/common": "~13.2.0",
-    "@angular/compiler": "~13.2.0",
-    "@angular/core": "~13.2.0",
-    "@angular/forms": "~13.2.0",
-    "@angular/material": "^13.2.3",
-    "@angular/platform-browser": "~13.2.0",
-    "@angular/platform-browser-dynamic": "~13.2.0",
-    "@angular/router": "~13.2.0",
+    "@angular/animations": "^16.2.10",
```

Comment:
Should all these Angular packages be in "dependencies"? I would put some of these packages to "peerDependencies" and if they're required for development also add them to "devDependencies".

I guess those packages should be moved:
```
"@angular/animations": "^16.2.10",
"@angular/common": "^16.2.10",
"@angular/core": "^16.2.10",
"@angular/forms": "^16.2.10",
"@angular/platform-browser": "^16.2.10",
"@angular/platform-browser-dynamic": "^16.2.10",
"@angular/router": "^16.2.10",
"rxjs": "~7.5.0"
```

Reply:
I will deal with the comments in separate tasks.